bookstore/tests/test_clone.py

```diff
@@ -119,28 +123,26 @@ def test_gen_template_params(self):
         assert expected == output
 
     def test_gen_template_params_base_url(self):
-        base_url_list = ['/my_base_url', '/my_base_url/', 'my_base_url/', 'my_base_url']
         expected = {
             'post_model': {'s3_bucket': 'hello', 's3_key': 'my_key'},
-            'clone_api_url': 'https://localhost:8888/my_base_url/api/bookstore/clone',
-            'redirect_contents_url': 'https://localhost:8888',
+            'clone_api_url': '/my_base_url/api/bookstore/clone',
+            'redirect_contents_url': '/my_base_url',
             'source_description': "'my_key' from the s3 bucket 'hello'",
         }
-        for base_url in base_url_list:
-            mock_app = Mock(
-                spec=Application,
-                ui_methods={},
-                ui_modules={},
-                settings={'jinja2_env': Environment(), "base_url": base_url},
-            )
-
-            success_handler = self.get_handler(
-                '/bookstore/clone?s3_bucket=hello&s3_key=my_key', app=mock_app
-            )
-            output = success_handler.construct_template_params(
-                s3_bucket="hello", s3_object_key="my_key"
-            )
-            assert expected == output
+        mock_app = Mock(
+            spec=Application,
+            ui_methods={},
+            ui_modules={},
+            settings={'jinja2_env': Environment(), "base_url": "/my_base_url/"},
```

Comment:
Nice, glad you have an explicit check with a `/` for `base_url` and a different one. 😄